cli/tests/e2e/targets/dependency_aware/poetry_trailing_newlines/poetry.lock

```diff
@@ -0,0 +1,1286 @@
+[[package]]
```

Comment:
I feel like the test file should be small so that the thing being tested is easy to see. Like 10 lines